client/components/containers/signup.js

```diff
@@ -3,14 +3,14 @@ import { connect } from 'react-redux';
 import createUser from '../../actions/createUser';
 import sendUserData from '../../actions/sendUserData';
```

Comment:
- Why sre your components named `.js` instead of `.jsx`
- Take care of the binary files showing on your repo